hardhat.config.ts

```diff
@@ -6,7 +6,6 @@ import "hardhat-gas-reporter"
 import "dotenv/config"
 import "solidity-coverage"
 import "hardhat-deploy"
-import "solidity-coverage"
```

Comment:
Why is this removed?

Reply:
Since Hardhat does some magical woowoo by injecting imports into the global scope (which can "pollute" global scope), I didnt find anything that was actually using this package in the scripts and tests.  Happy to reinstate, but since HH injects all sorts o f things I felt maybe removing it is one less then for your users to analyse /evaluate when studying this config file.   So purely hygiene -- but on the basis that this package does not "appear" to be used.  Is it?

Reply:
No I think you're right.